### DISCUSSION TITLE
Zora

### DISCUSSION THREAD

**Discussion by matheusfm** [aquasecurity/trivy#5489] [Adopters]
### [Optional] How do you use Trivy?

Hello everyone!

Trivy has been integrated in [Zora OSS](https://zora-docs.undistro.io/) as a vulnerability scanner plugin for Kubernetes clusters!

Zora is an open-source solution that scans your clusters with multiple plugins at scheduled times. For more details, please refer to our [documentation](https://zora-docs.undistro.io/).

Zora plugins, which are CLI tools, are responsible for scanning Kubernetes clusters. In this integration, the command `trivy k8s --scanners=vuln ...` is used.

To know more about how plugins work, [visit our plugin documnetation](https://zora-docs.undistro.io/latest/plugins/) and the [Trivy plugin page](https://zora-docs.undistro.io/latest/plugins/trivy/).

We're thrilled to have Trivy as part of our Zora OSS toolkit!

### [Optional] Which targets are you scanning with Trivy?

- [ ] Container Image
- [ ] Filesystem
- [ ] Git Repository
- [ ] Virtual Machine Image
- [X] Kubernetes
- [ ] AWS
- [ ] SBOM

### [Optional] What kind of issues are scanning with Trivy?

- [ ] Software Bill of Materials (SBOM)
- [X] Known vulnerabilities (CVEs)
- [ ] IaC issues and misconfigurations
- [ ] Sensitive information and secrets
- [ ] Software licenses

**Comment by knqyf263** [comment]
Thanks for sharing! It looks cool.

**Comment by AnaisUrlichs** [comment]
Hi there, would you like to add Zora to our ecosystem page? https://aquasecurity.github.io/trivy/v0.48/ecosystem/security/

**Reply by matheusfm** [reply]
Hi @AnaisUrlichs 
Zora is included in "Production" category of ecosystem page: https://aquasecurity.github.io/trivy/v0.48/ecosystem/prod/
Do you think this category fits better? If yes we can do it.

**Reply by AnaisUrlichs** [reply]
what do you think of moving it to "Security Management" https://aquasecurity.github.io/trivy/v0.48/ecosystem/security/

**Reply by matheusfm** [reply]
what are the criteria for being in each?
